mobile/src/screens/marketplace.js

```diff
@@ -564,9 +565,71 @@ class MarketplaceScreen extends Component {
     updateExchangeRate(this.state.fiatCurrency[1], 'DAI')
   }
 
-  onWebViewNavigationStateChange = state => {
+  _openDeepLinkUrlAttempt = async (
+    interceptUrlPredicate,
+    makeUrl,
+    timeControlVariableName
+  ) => {
+    // non interceptable url
+    if (!interceptUrlPredicate()) return
+
+    const url = makeUrl()
+    if (Linking.canOpenURL(url)) {
```

Comment:
Ahh I guess this isn't true if they don't have Facebook/Twitter?

Reply:
yeah it just opens the thing in the native browser as it does right now.